ServerCore/Pages/Events/Players.cshtml

```diff
@@ -1,64 +1,154 @@
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Admins[0].EmployeeAlias)
+            </th>
+            <th>
+            </th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+        @foreach (var item in Model.Admins)
+        {
+            <tr>
+                <td>
+                    @Html.DisplayFor(modelItem => item.Name)
+                </td>
+                <td>
+                    @Html.DisplayFor(modelItem => item.Email)
+                </td>
+                <td>
+                    @Html.DisplayFor(modelItem => item.EmployeeAlias)
+                </td>
+                <td>
+                    <a asp-page-handler="RemoveAdmin" asp-route-userId=@item.ID>Remove</a>
+                </td>
+                <td></td>
+            </tr>
+        }
+    </tbody>
+</table>
+
+<h4>All admin e-mails</h4>
+
+<p>@Model.AdminEmails</p>
+
+<h3>Authors</h3>
+<a asp-page="/Events/AddAdminOrAuthor" asp-route-addAdmin=false>Add author</a>
 
 <table class="table">
     <thead>
         <tr>
             <th>
-                Player @Html.DisplayNameFor(model => model.Players[0].Member.ID)
+                @Html.DisplayNameFor(model => model.Authors[0].Item1.Name)
             </th>
             <th>
-                Player @Html.DisplayNameFor(model => model.Players[0].Member.Name)
+                @Html.DisplayNameFor(model => model.Authors[0].Item1.Email)
             </th>
             <th>
-                Player @Html.DisplayNameFor(model => model.Players[0].Member.Email)
+                @Html.DisplayNameFor(model => model.Authors[0].Item1.EmployeeAlias)
             </th>
             <th>
-                Team @Html.DisplayNameFor(model => model.Players[0].Team.ID)
+                Puzzle count
             </th>
             <th>
-                Team @Html.DisplayNameFor(model => model.Players[0].Team.Name)
             </th>
             <th></th>
         </tr>
     </thead>
     <tbody>
-        @foreach (var item in Model.Players)
+        @foreach (var item in Model.Authors)
         {
             <tr>
                 <td>
-                    @Html.DisplayFor(modelItem => item.Member.ID)
+                    @Html.DisplayFor(modelItem => item.Item1.Name)
                 </td>
+                <td>
+                    @Html.DisplayFor(modelItem => item.Item1.Email)
+                </td>
+                <td>
+                    @Html.DisplayFor(modelItem => item.Item1.EmployeeAlias)
+                </td>
+                <td>
+                    @Html.DisplayFor(modelItem => item.Item2)
+                </td>
+                <td>
+                    <a asp-page-handler="RemoveAuthor" asp-route-userId=@item.Item1.ID>Remove</a>
+                </td>
+                <td></td>
+            </tr>
+        }
+    </tbody>
+</table>
+
+<h4>All author e-mails</h4>
+
+<p>@Model.AuthorEmails</p>
+
+<h3>Players</h3>
```

Comment:
Consider having this be separate or optional to avoid putting a ton of content on this page when it's not wanted.

Reply:
Will do separately - good call :)